cratedb_toolkit/cmd/tail/cli.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import sys
+
+import click
+
+from cratedb_toolkit.cmd.tail.main import TableTailer
+from cratedb_toolkit.model import TableAddress
+from cratedb_toolkit.util import DatabaseAdapter
+from cratedb_toolkit.util.cli import boot_click
+
+logger = logging.getLogger(__name__)
+
+cratedb_sqlalchemy_option = click.option(
+    "--cratedb-sqlalchemy-url", envvar="CRATEDB_SQLALCHEMY_URL", type=str, required=False, help="CrateDB SQLAlchemy URL"
+)
+
+
+@click.command()
+@cratedb_sqlalchemy_option
+@click.option(
+    "--lines", "-n", type=int, required=False, default=10, help="Displays n last lines of the input. Default: 10"
+)
+@click.option("--format", "format_", type=str, required=False, help="Select output format. Default: log / jsonl")
+@click.option("--follow", "-f", is_flag=True, required=False, help="Follow new records added, by polling the table")
+@click.option(
+    "--interval", "-i", type=float, required=False, help="When following the tail, poll each N seconds. Default: 0.5"
+)
+@click.option("--verbose", is_flag=True, required=False, help="Turn on logging")
+@click.option("--debug", is_flag=True, required=False, help="Turn on logging with debug level")
```

Comment:
Any reason not to use `click` default option settings for all default values instead of later on inside the code?

Reply:
If the defaults would be applied to click, they wouldn't apply when using ctk as a Python API in your own programs.